src/plugins/conceptPlugin.js

```diff
@@ -95,14 +96,18 @@ export default function createConceptPlugin(options = {}) {
       undefined,
       { concept: true }
     );
-
     return render(
       <Notion
-        id={`notion_id_${id}`}
+        id={`notion_id_${id}_${locale}`}
         ariaLabel={t(locale, 'concept.showDescription')}
         title={title}
+        customCSS={css`
+          left: 0 !important;
```

Comment:
har du sjekket at det ikke funker uten important?

`Always look for a way to use specificity before even considering !important
` - https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#the_!important_exception

Reply:
Det vil ikke fungere uten !important fordi det er mange media queries der.

En alternativ løsning er å sette `{left: 0; margin-left: 0;` med alle de samme media queries. Jeg valgte å ikke gjøre dette fordi det potensielt kan komme flere media queries eller de kan endre seg, men kanskje det likevel er det riktige å gjøre for å unngå !important.

Reply:
> En alternativ løsning er å sette {left: 0; margin-left: 0; med alle de samme media queries
skjønte ikke helt hva du mente med dette?

Reply:
CSS som jeg ønsker å overstyre ser slik ut:
```
  ${mq.range({ from: breakpoints.tablet })} {
    max-width: 500px;
    left: 50%;
    margin-left: -250px;
  }
  ${mq.range({ from: breakpoints.desktop })} {
    max-width: 720px;
    margin-left: -360px;
    left: 50%;
  }
```

Dersom jeg sender inn css'en slik det er nå, men uten important vil dette ha mindre prioritet enn media queryene ovenfor.

Alternativ 1 fungerer:
```
left: 0 !important;
margin-left: 0 !important;
width: 100% !important;
```

Alternativ 2 fungerer ikke:
```
left: 0;
margin-left: 0;
width: 100%;
```

Alternativ 3 burde fungere. Dette er vel egentlig en bedre løsning enn alternativ 1 siden man slipper !important:
```
  ${mq.range({ from: breakpoints.tablet })} {
left: 0;
margin-left: 0;
width: 100%;
  }
  ${mq.range({ from: breakpoints.desktop })} {
left: 0;
margin-left: 0;
width: 100%;
  }
```

Reply:
enig i den siste der, alt. 3, så lenge det fungerer 👏 Da tvinger vi ikke gjennom noe i css hierarkiet/cascading'en